androidApp/src/main/java/com/mbta/tid/mbta_app/android/pages/NearbyTransitPage.kt

```diff
@@ -118,13 +121,16 @@ fun NearbyTransitPage(
 
     val viewModel: NearbyTransitTabViewModel = viewModel()
 
-    val currentNavEntry: SheetRoutes? =
-        viewModel.currentNavEntry.collectAsState(initial = null).value
-    val previousNavEntry: SheetRoutes? =
-        viewModel.previousNavEntry.collectAsState(initial = null).value
```

Comment:
Keeping the `currentNavEntry` on the VM was causing lag issues when applying filters switching between direction. Now that I'm have a better understanding of the relationship between CurrentNavBackStackEntry & SheetRoutes, we can just convert from one to the other on the fly.